2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,51 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+let recipe1 = {
+  Title: "Fig Cake",
+  Serving: 8,
+  Ingredients: ["Fig", "Walnut", "Flour", "Sugar", "Milk", "Egg"],
+};
+
+let recipe2 = {
+  Title: "Menemen",
+  Serving: 4,
+  Ingredients: ["Egg", "Tomato", "Pepper", "Olive Oil", "Salt"],
+};
+
+let recipe3 = {
+  Title: "Chocolate Cookie",
+  Serving: 5,
+  Ingredients: ["Egg", "Sugar", "Butter", "Hazelnut", "Flour"],
+};
+
+let recipe4 = {
+  Title: "Red Lentil Soup",
+  Serving: 6,
+  Ingredients: ["Red Lentil", "Salt", "Olive Oil", "Carrot", "Potato", "Onion"],
+};
+
+let recipe5 = {
+  Title: "Rice",
+  Serving: 3,
+  Ingredients: ["Rice", "Salt", "Butter", "Olive Oil", "Water"],
+};
+
+let recipes = [recipe1, recipe2, recipe3, recipe4, recipe5];
+
+recipes.forEach((recipe) => {
```

Comment:
An interesting version :) I like that you iterate over object properties, but think it makes the code a bit difficult to understand as you depend on order of your properties (`index === 0`, etc).

Maybe you can achieve the same results without nested forEach:

```javascript
recipes.forEach((recipe) => {
  console.log(recipe.Title);
  console.log("Serves:" + recipe.Serving);
  recipe.Ingredients.forEach((ingredient) => {
    console.log(ingredient);
  });
});
```